addons/mail/tests/test_mail_render.py

```diff
@@ -210,6 +222,15 @@ def test_render_template_inline_template(self):
             )[partner.id]
             self.assertEqual(rendered, expected)
 
+    @users('employee')
+    def test_render_template_local_links(self):
+        for source, expected in zip(self.base_local_links_template_bits, self.base_rendered_local_links):
+            rendered = self.env['mail.render.mixin']._replace_local_links(
+                source,
+                'https://www.example.com',
```

Comment:
Why giving ``'https://www.example.com'`` as base_url ?

Reply:
Why not?